test-interface/src/main/scala/org/scalajs/testinterface/HTMLRunner.scala

```diff
@@ -242,14 +250,19 @@ protected[testinterface] object HTMLRunner extends js.JSApp {
       def log(msg: String, clss: String): dom.Element =
         body.newElement(clss = s"log $clss", text = msg, tpe = "pre")
 
+      def setNextSibling(that: TestBox): Unit = {
+        this.box.insertAdjacentElement("afterEnd", that.box)
```

Comment:
[On MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML) the doc says to use `"afterend"` all-lowercase. I suggest we do not tempt fate with alternative casing.